From: Michael Davidsaver <mdavidsaver@gmail.com>
Date: Wed, 19 Jul 2017 14:21:46 +0200
Subject: disable epicsExceptionTest

not needed with modern c++ and fails to build
with modern g++
---
 src/libCom/test/Makefile              | 5 -----
 src/libCom/test/epicsRunLibComTests.c | 2 --
 2 files changed, 7 deletions(-)

diff --git a/src/libCom/test/Makefile b/src/libCom/test/Makefile
index 51ee39f..2dac768 100755
--- a/src/libCom/test/Makefile
+++ b/src/libCom/test/Makefile
@@ -166,11 +166,6 @@ epicsAtomicTest_SRCS += epicsAtomicTest.cpp
 testHarness_SRCS += epicsAtomicTest.cpp
 TESTS += epicsAtomicTest
 
-TESTPROD_HOST += epicsExceptionTest
-epicsExceptionTest_SRCS += epicsExceptionTest.cpp
-testHarness_SRCS += epicsExceptionTest.cpp
-TESTS += epicsExceptionTest
-
 TESTPROD_HOST += macDefExpandTest
 macDefExpandTest_SRCS += macDefExpandTest.c
 testHarness_SRCS += macDefExpandTest.c
diff --git a/src/libCom/test/epicsRunLibComTests.c b/src/libCom/test/epicsRunLibComTests.c
index 0f10dbf..66389b4 100644
--- a/src/libCom/test/epicsRunLibComTests.c
+++ b/src/libCom/test/epicsRunLibComTests.c
@@ -23,7 +23,6 @@ int epicsEllTest(void);
 int epicsEnvTest(void);
 int epicsErrlogTest(void);
 int epicsEventTest(void);
-int epicsExceptionTest(void);
 int epicsExitTest(void);
 int epicsMathTest(void);
 int epicsMessageQueueTest(void);
@@ -75,7 +74,6 @@ void epicsRunLibComTests(void)
     runTest(epicsEnvTest);
     runTest(epicsErrlogTest);
     runTest(epicsEventTest);
-    runTest(epicsExceptionTest);
     runTest(epicsMathTest);
     runTest(epicsMessageQueueTest);
     runTest(epicsMMIOTest);
